### DISCUSSION TITLE
ERR_TOO_MANY_REDIRECTS in public website with Filament PHP CMS

### DISCUSSION THREAD

**Discussion by daniloquispe** [filamentphp/filament#13490] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

Hello:

I'm creating a dynamic website with a CMS using Laravel 11 and Filament PHP 3.2. I use Laravel server for local development and testing.

Public website has, for now, some simple pages created with standard controllers and Blade views/components (no LiveWire, Inertia nor other frontend engines). Public routes for these pages were manually added to routes/web.php as usual.

My Filament instance has only 1 panel, called "CMS".

* When I open my public homepage (http://localhost:8000) in my browser, website works fine.
* When I open and login into the CMS made with Filament (http://localhost:8000/cms), CMS works fine.
* But, when I open and login into the CMS in a browser tab, and then try to open a public page in a second tab, my browser shows an error `ERR_TOO_MANY_REDIRECTS` in second tab and public pages stop loading (Filament pages still work fine).

Now, my questions are:

* Why does this occur?
* Can I fix it?
* If I have a Filament PHP project, can I add my own pages (controllers, views and routes) outside any Filament panel? Or do I need to extract my public website to a non-Filament Laravel project?

Thanks in advance.

**Comment by ghost** [comment]
Did you solve it?

**Comment by HoceineEl** [comment]
@daniloquispe in case you're still building the public site side, I've been working on FilamentCraft, it's a website builder that plugs into a Filament panel: themes, sections, visual editor, SEO, multi-language. Might save you some of the plumbing.

Demo (no signup): https://demo.filamentcraft.dev/launch/admin
